.github/workflows/ci.yaml

```diff
@@ -15,14 +15,10 @@ jobs:
 
     - name: Install dependencies
       run: |
-        sudo apt-get install python3-setuptools
-        sudo pip3 install black flake8
+        sudo apt install tox
```

Comment:
I'd probably recommend doing `pip install tox` over `apt install`, the version that is packaged in the archives is quite old.